Model/Config.php

```diff
@@ -44,6 +44,12 @@ public function isSkipFraudDetection()
 
     public function isTestMode()
     {
+        $ip = isset($_SERVER['HTTP_CLIENT_IP']) ? $_SERVER['HTTP_CLIENT_IP'] : isset($_SERVER['HTTP_X_FORWARDED_FOR']) ? $_SERVER['HTTP_X_FORWARDED_FOR'] : $_SERVER['REMOTE_ADDR'];
+        $ipconfig = $this->store->getConfig('payment/paynl/testipaddress');
```

Comment:
Kun je nog even een check toevoegen of $ip toch echt wel gevuld is en een IP is. Stel dat deze toch leeg is, dan terug vallen op oude werkwijze.